include/battle.h

```diff
@@ -678,6 +677,8 @@ struct BattleStruct
     bool8 trainerSlideBeforeFirstTurnMsgDone;
     u32 aiDelayTimer; // Counts number of frames AI takes to choose an action.
     u32 aiDelayFrames; // Number of frames it took to choose an action.
+    u8 faintedMonCount[NUM_BATTLE_SIDES];
```

Comment:
You can use these fields : https://github.com/rh-hideout/pokeemerald-expansion/blob/master/include/battle.h#L369-L373